src/main/scala/mesosphere/marathon/ZookeeperConf.scala

```diff
@@ -12,23 +13,19 @@ trait ZookeeperConf extends ScallopConf {
   private val zkNode = """[^/]+"""
   private val zkURLPattern = s"""^zk://(?:$userAndPass@)?($hostAndPort(?:,$hostAndPort)*)(/$zkNode(?:/$zkNode)*)$$""".r
 
-  @Deprecated
-  lazy val zooKeeperHostString = opt[String]("zk_hosts",
-    descr = "[DEPRECATED use zk] The list of ZooKeeper servers for storing state",
-    default = Some("localhost:2181"))
-
-  @Deprecated
-  lazy val zooKeeperPath = opt[String]("zk_state",
```

Comment:
You should remove this from the Docs as well (see rest-api.md and developing-vm.md)